ce/api/util.py

```diff
@@ -248,22 +245,21 @@ def search_for_unique_ids(
     )
 
     matching_cell_methods = filter_by_climatological_statistic(
-        [r[0] for r in cell_methods], climatological_statistic,
+        [r[0] for r in cell_methods],
+        climatological_statistic,
```

Comment:
Aside: Black respects a trailing comma by placing each item on a separate line, regardless of whether they would all fit on one line. You may or may not want it like this. No big deal.